build.gradle

```diff
@@ -730,6 +730,7 @@ subprojects {
 
     runtimeOnly 'org.apache.logging.log4j:log4j-core'
     runtimeOnly 'org.apache.logging.log4j:log4j-slf4j-impl'
+    runtimeOnly 'org.apiguardian:apiguardian-api'
```

Comment:
I suspect we can add this as a testRuntimeOnly if it's required by junit.  Adding it to `runtimeOnly` will cause it to be shipped with Teku which I don't think we want.